packages/apidom-ls/src/services/validation/linter-functions.ts

```diff
@@ -1009,9 +1011,35 @@ export const standardLinterfunctions: FunctionItem[] = [
     functionName: 'apilintOpenAPIPathTemplateValid',
     function: (element: Element) => {
       if (isStringElement(element)) {
-        return true;
+        const pathItemElement = (element.parent as MemberElement).value as ObjectElement;
+
+        if (pathItemElement.length === 0) {
+          return true;
+        }
+
+        const pathTemplateResolveParams: { [key: string]: string } = {};
+
+        pathItemElement.forEach((operation) => {
```

Comment:
`Path Item Object` can contain additional fixed fields like `servers`, `summary` or `description`. Which means the callback will receive `StringElement` (summary or description fields) or `ArrayElement` (servers field), and not just `OperationElement`. That's why I did following filtering in my "semantic code template":

```ts
        // we're asserting OperationElement polymorphically as this function has to support all OpenAPI versions
        const operationElements = Array.from(
          pathItemElement.filter((elm: Element) => elm.element === 'operation').map((elm) => elm),
        );
```